openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/EntityDataConsumer.spec.ts

```diff
@@ -75,22 +75,25 @@ entities.forEach((EntityClass) => {
       await entity.visitEntityPage(page);
     });
 
-    test('User as Owner Add, Update and Remove', async ({ page }) => {
-      test.slow(true);
+    // Running following 2 tests serially since they are dependent on each other
+    test.describe.serial('Owner permission tests', () => {
+      test('User as Owner Add, Update and Remove', async ({ page }) => {
+        test.slow(true);
+
+        const OWNER1 = EntityDataClass.user1.getUserName();
+        const OWNER2 = EntityDataClass.user2.getUserName();
+        const OWNER3 = EntityDataClass.user3.getUserName();
+        await entity.owner(page, [OWNER1, OWNER3], [OWNER2], undefined, false);
+      });
 
-      const OWNER1 = EntityDataClass.user1.getUserName();
-      const OWNER2 = EntityDataClass.user2.getUserName();
-      const OWNER3 = EntityDataClass.user3.getUserName();
-      await entity.owner(page, [OWNER1, OWNER3], [OWNER2], undefined, false);
-    });
+      test('No edit owner permission', async ({ page }) => {
+        await page.reload();
+        await page.waitForSelector('[data-testid="loader"]', {
+          state: 'detached',
+        });
 
-    test('No edit owner permission', async ({ page }) => {
-      await page.reload();
-      await page.waitForSelector('[data-testid="loader"]', {
-        state: 'detached',
+        await expect(page.getByTestId('edit-owner')).not.toBeAttached();
```

Comment:
instead of serial can we use test.step?
i think nested describe can give issue here.

Reply:
Since tests have different functionalities, it is better to keep them separate.